### ISSUE TITLE
Show a toast notification when you successfully log in (and possibly some other things)

### ISSUE THREAD

**Issue by nbudin** [neinteractiveliterature/intercode#4925]
Having successfully logged into the site isn't particularly obvious to sighted users, and much less so to screenreader users.  We should have a more obvious way to show that you did it.  I'm thinking a toast notification; let's make sure this is fully accessible to screenreaders when doing it.
